manifests/metering-config/recommended-resource-limits.yaml

```diff
@@ -0,0 +1,149 @@
+apiVersion: metering.openshift.io/v1alpha1
+kind: Metering
+metadata:
+  name: "operator-metering"
+spec:
+  reporting-operator:
+    spec:
+      config:
+        # reporting-operator does the work of importing Metrics in chunks of 5
+        # minutes (by default), every 5 minutes.
+        # This only needs to be adjusted if your cluster is really large, and 5
+        # minutes of data is more than a few hundred metrics per time series.
+        # This comes up when you have hundreds of namespaces and pods.
+        promsumChunkSize: "5m"
+        promsumPollInterval: "5m"
+        # This is the resolution of the metrics. Setting this higher reduces
+        # granularity of the data we import, but also reduces the number of
+        # metrics returned per query.
+        promsumStepSize: "60s"
+
+      # Increasing memory to 500Mi allows reporting-operator to keep more
```

Comment:
this should probably be the same as the number listed in the actual config (standardize to either 500 or 512; elsewhere you mainly use multiples of 5 so I'd recommend 500)

Reply:
Whoops. I was messing with the different unit suffixes. Good catch.